Update Sentinel documentation (#1277)
MicroEJ/docs: SDK6UserGuide/licenses_sentinel.rst

- Set ``DYLD_LIBRARY_PATH`` variable with command ``export DYLD_LIBRARY_PATH=<your_directory>:$DYLD_LIBRARY_PATH``.
  This modification has to be setup at session startup  (e.g: using ``.bashrc`` file) to ensure that OS is properly configured before running the SDK.

Then you can continue with the :ref:`Remote Floating License Server <add_remonte_floating_license_server>` section.

> .. _add_remonte_floating_license_server:

Reviewer: ```suggestion
.. _add_remote_floating_license_server:
```

Add Remote Floating License Server
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> **With a browser**

Reviewer: ```suggestion
**Fron Sentinel Admin Control Center**
```

.. note::

>    If you use WSL (Windows Subsystem for Linux) localhost may point to Windows and not to your WSL instance. On WSL terminal you can launch ``hostname -I`` command, copy the first IP and use it insteal of ``localhost`` in your browser.

Reviewer: ```suggestion
   If you use WSL (Windows Subsystem for Linux) ``localhost`` refers to your Windows localhost and not to your WSL instance. On WSL terminal, launch ``hostname -I`` command, copy the first IP address and use it instead  of ``localhost`` in your browser (e.g. http://172.30.106.171:1947/).
```
am I right ?

>    If you use WSL (Windows Subsystem for Linux) localhost may point to Windows and not to your WSL instance. On WSL terminal you can launch ``hostname -I`` command, copy the first IP and use it insteal of ``localhost`` in your browser.

Reviewer: However, this does not work on my local env:
![image](https://github.com/user-attachments/assets/9d52a025-c68b-4eea-bdec-d1d3529607e8)

Reviewer: thanks,
successfully works with `accremote = 1`

> **With command line**

Reviewer: ```suggestion
**From Command Line**
```

> - On Linux update or create the file ``hasp_37102.ini`` in ``/etc/hasplm`` or ``~/.hasplm`` (if you have not installed RTE as root). For Windows edit ``%CommonProgramFiles(x86)%\Aladdin Shared\HASP\hasplm.ini`` file. For both OS add these lines to the ini file:

Reviewer: ```suggestion
- On Linux: update or create the file ``hasp_37102.ini`` in ``/etc/hasplm`` or ``~/.hasplm`` (if you have not installed RTE as root). 
- On Windows: edit ``%CommonProgramFiles(x86)%\Aladdin Shared\HASP\hasplm.ini`` file. 

Then add to the ``.ini`` file the following lines:
```

   
   .. code-block::
      
      [REMOTE]
      broadcastsearch = 0
      serversearchinterval = 30
      serveraddr = <license_server_IP>

> - Restart the service.

Reviewer: Explain how to restart the service
- Linux: `sudo service aksusbd status`
- Windows and MacOS : How do we do that for

Author: I've added explanation for Windows & mac.

Running in a container
~~~~~~~~~~~~~~~~~~~~~~

If you want to configure a CI (Continuous integration) runner you can follow one of these two solutions:

> 1. Create a Docker image with the RTE installed inside, see :ref:`Installation for Linux <setup_sentinel_developer_workstation>`

Reviewer: ```suggestion
- Either create a Docker image with the RTE installed inside, see :ref:`Installation for Linux <setup_sentinel_developer_workstation>` section.
```

> 2. Or install and configure the RTE on the host and run the Docker container with these options:

Reviewer: ```suggestion
- Or install and configure the RTE on the host and run the Docker container with these options:
```
